cmd/view.go

```diff
@@ -48,24 +53,74 @@ func viewCmd(ctx *cli.Context) error {
 		return err
 	}
 
-	verbose := ctx.Bool("verbose")
-	if verbose {
-		fmt.Printf("Credential path: %s\n\n", path)
+	format := ctx.String("format")
+	if ctx.Bool("verbose") {
+		format = "verbose"
+	}
```

Comment:
`--verbose` should take precedence, I had it throw a usage error if you used both but the help explicitly states `--verbose` is the same as `--format verbose`.